keyboards/leafcutterlabs/bigKNOBv2/bigKNOBv2.c

```diff
@@ -0,0 +1,5 @@
+#include "bigKNOBv2.h"
+
+void matrix_init_kb(void) {
+	matrix_init_user();
+}
```

Comment:
This duplicates the default function, so you can remove it here.

https://github.com/qmk/qmk_firmware/blob/ba7f52aaeba878544973952ca3fcce27ab4bb92d/quantum/matrix_common.c#L21

```suggestion
```